package-lock.json

```diff
@@ -1,18158 +1,7 @@
 {
   "name": "xrpl.js",
-  "lockfileVersion": 2,
```

Comment:
Was it intentional to delete 18,000 lines from the package-lock? (maybe a rebase is required?)

Reply:
before making my latest changes I have merged the develop branch into my "secret numbers" branch. When I check the history of the package-lock.json file of the develop branch, there are also some big changes which might have caused this:

https://github.com/XRPLF/xrpl.js/commit/9f19d771d24fa186ae23d586f380725f53a59059#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519

https://github.com/XRPLF/xrpl.js/commit/2b424276344b2aa8b8b76d621500f4d9e1436663#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519

Reply:
I believe this is happening because you're using a different version of npm. I'm on v7.20.3.

Reply:
I just updated to npm 7.24.x and the package-lock.json was not changed after "npm ci"
I still  believe it is because I have merged the develop branch into my branch

Reply:
`npm ci` doesn't change the package-lock, instead it reads from the package-lock for a faster install with deterministic package resolution. if the package-lock needs to be written, as in this case, you need `npm i`

Reply:
check out my documentation in `CONTRIBUTING.md` for how to add/remove packages in this repo